shoop/admin/modules/orders/static_src/create/view/confirm.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * This file is part of Shoop.
+ *
+ * Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+function renderHeaders() {
+    return  m("tr", [
+        m("th", gettext("SKU")),
+        m("th", gettext("Text")),
+        m("th", gettext("Quantity")),
+        m("th", gettext("Unit Price")),
+        m("th", gettext("Discount amount")),
+        m("th", gettext("Total (exclude taxes)")),
+        m("th", gettext("Tax percent")),
+        m("th", gettext("Total"))
+    ]);
+}
+
+function renderLines(lines) {
+    return _(lines).map((line) => {
+        return m("tr", [
+            m("td", line.sku),
+            m("td", line.text),
+            m("td", line.quantity),
+            m("td", line.unitPrice),
+            m("td", line.discountAmount),
+            m("td", line.taxlessTotal),
+            m("td", line.taxPercentage),
+            m("td", line.taxfulTotal)
+        ]);
+    }).compact().value();
```

Comment:
`compact()` shouldn't be necessary (unless there can somehow be `null`s in the `lines` array?)